### ISSUE TITLE
Suggestion/Question: Is it possible to make the batch editor do individual tasks based on individual filters?

### ISSUE THREAD

**Issue by ReZeroEmiria** [kwsch/PKHeX#1521]
I'm editing through my old collection of Pokemon from previous generations and hyper training them. I tried the following on the batch editor:

!IV_HP=31
.HT_HP=True
!IV_ATK=0
!IV_ATK=1
!IV_ATK=31
.HT_ATK=True
!IV_DEF=31
.HT_DEF=True
!IV_SPA=31
.HT_SPA=True
!IV_SPD=31
.HT_SPD=True
!IV_SPE=31
.HT_SPE=True

What I want this script to do is to check through each Pokemon and Hyper Train each stat if a stat isn't already 31, the exception being the ATK IV because I'd want that to stay 0/1 if I have a low atk IV for competitive purposes.

After reading the batch editor notes, I saw that this would make it filter out each IV, and this script would only apply to pokemon with NO IV's of 31 as opposed to what I wanted.

I'm not sure if it's possible to set it so that multiple scripts can be run individually. Like

!IV_HP=31
.HT_HP=True

Then

!IV_ATK=0
!IV_ATK=1
!IV_ATK=31
.HT_ATK=True

Then

!IV_DEF=31
.HT_DEF=True

Then

!IV_SPA=31
.HT_SPA=True

Then

!IV_SPD=31
.HT_SPD=True

Then

!IV_SPE=31
.HT_SPE=True

So that It will search for all Pokemon without 31 in HP, set them to Hyper Trained, then ATK if it's not 0/1/31 it will set it to Hyper Trained, then DEF if not 31 and so on...

If there is a way to do this in one script without having to do this individually please let me know, I looked at the batch editor post on projectpokemon I think and I couldn't find anything on this. Thank You!

**Comment by kwsch** [comment]
Probably better to have a `HyperTrainFlags=$suggest`

https://github.com/kwsch/PKHeX/blob/0120263bd3d56603c703d91521ba62a3a1a6e61b/PKHeX.WinForms/Subforms/PKM%20Editors/BatchEditor.cs#L492

**Comment by ReZeroEmiria** [comment]
When using this in the batch editor on some Pokemon I get:

Modified 0/6 files.
6 files ignored due to an internal error.

**Comment by kwsch** [comment]
It's not currently in the program; I was listing it as a possible addition to resolve the issue (auto-hyper training pkm is useful)

**Comment by ReZeroEmiria** [comment]
I see, I would still suggest adding a way for multiple actions to be put under different filters maybe adding a comma at the end of a line?

!IV_HP=31
.HT_HP=True ,

!IV_ATK=0
!IV_ATK=1
!IV_ATK=31
.HT_ATK=True

This would tell the program to search through all Pokemon, if HP IV is not 31, then set Hyper Training to True 

(Comma here would indicate 'THEN')

If ATK IV is not 0,1, or 31 then set Hyper Training to True

The reason I suggest this is because a suggested Hyper Training command might not always be right as some people will want a lower ATK IV to reduce confusion/foul play etc damage or a lower Speed IV for Trick Room. In some cases even hindering natures will want a higher ATK IV for mainly special attackers which only have the Physical U-Turn to leave battle and cannot afford to lower another stat of theirs.

I came across the speed issue while I continued going through my Pokemon and I figured that adding a suggested option might not appeal to everyone for the reasons above, I feel adding a way to execute multiple commands through batch editor will make it far more useful.

Also, this is a different issue but as I went through a Japanese Faraway Island Mew from Pokemon Emerald, I noticed that I could put an English OT on it. For legality's sake, I did some research into how many Characters were allowed in the Japanese Version game. The answer I came to was 5. So, maybe add an extra field for that so that Pkhex flags a Pokemon for OT being too long for the Japanese Version. 

In this case I can't set OT to "Emilia" but "エミリア" will do~
